labellines/core.py

```diff
@@ -96,7 +100,12 @@ def ensure_float(value):
         fraction = 0.5
     else:
         fraction = (ensure_float(x) - xfa) / (xfb - xfa)
-    y = ya + (yb - ya) * fraction + yoffset
+
+    if yoffset_logspace:
+        y = ya + (yb - ya) * fraction
+        y = 10 ** (np.log10(y) + yoffset)
+    else:
+        y = ya + (yb - ya) * fraction + yoffset
```

Comment:
```suggestion
    if yoffset_logspace:
        y = ya + (yb - ya) * fraction
        y *= 10 ** yoffset
    else:
        y = ya + (yb - ya) * fraction + yoffset
```

Reply:
One option is to get rid of `yoffset_logspace`, and make the `yoffset` in log space if the y-axis is log space. I can't think of a time you would want a linear `yoffset` when the y axis is log.
```suggestion
    if ax.get_yscale() == 'log':
        y = ya + (yb - ya) * fraction
        y *= 10 ** yoffset
    else:
        y = ya + (yb - ya) * fraction + yoffset
```